terraform/ec2/main.tf

```diff
@@ -222,6 +222,7 @@ resource "null_resource" "start_collector" {
   provisioner "remote-exec" {
     inline = [
       local.ami_family["install_command"],
+      local.selected_ami["family"] == "amazon_linux" ? "sudo chmod 777 /opt/aws/aws-otel-collector/etc/.env && sudo echo 'SAMPLE_APP_HOST=${aws_instance.sidecar.public_ip}' >> /opt/aws/aws-otel-collector/etc/.env && sudo echo 'SAMPLE_APP_PORT=${module.common.sample_app_lb_port}' >> /opt/aws/aws-otel-collector/etc/.env" : "echo \"not amazon linux\"",
```

Comment:
can you configure it as a parameters in the ami family somewhere? like [here](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/ec2_setup/amis.tf#L3), i'm not sure if you want to support windows because the soaking test will also cover windows

Reply:
I tried that before, but it does not allow the use of the variable that I require

Reply:
As for windows support I can add that in, I was under the assumption that only amazon linux is used for perf/soaking tests